### DISCUSSION TITLE
Dependency vs devDependency

### DISCUSSION THREAD

**Discussion by szulcus** [victorgarciaesgi/nuxt-typed-router#78] [General]
Why does the documentation say to install this package as a dependency instead of devDependency? I found that devDependency works for me as well and this seems to be a better solution. Is there some greater significance that I haven't thought of?

**Comment by victorgarciaesgi** [comment]
I may have been suited for `devDependencies` before, but now that the module generate runtime utils like `useRouter`, `definePageMeta`, `useLocalePath` etc..
It will not change the way the app builds, but it can depends on where you build it (ex: Vercel/Netlify can do optimization based on this)

**Reply by szulcus** [reply]
Ok, thank you!
